scripts/generate_test_certs.sh

```diff
@@ -71,8 +71,8 @@ ${CERTSTRAP} --depot-path "${depot_path}" sign api_public --CA autoscaler-ca --y
 # scheduler certificate
 ${CERTSTRAP} --depot-path "${depot_path}" request-cert --passphrase '' --domain scheduler --ip 127.0.0.1
 ${CERTSTRAP} --depot-path "${depot_path}" sign scheduler --CA autoscaler-ca --years "20"
-openssl pkcs12 -export -in "${depot_path}"/scheduler.crt -inkey "${depot_path}"/scheduler.key -out "${depot_path}"/scheduler.p12 -name scheduler -password pass:123456
-keytool -importcert -alias autoscaler -file "${depot_path}"/autoscaler-ca.crt -keystore "${depot_path}"/autoscaler.truststore -storeType pkcs12 -storepass 123456 -noprompt
+#openssl pkcs12 -export -in "${depot_path}"/scheduler.crt -inkey "${depot_path}"/scheduler.key -out "${depot_path}"/scheduler.p12 -name scheduler -password pass:123456
```

Comment:
Should we just remove it ?

Reply:
Done in 367329f4bec4599a92d25f500f5ae0590c548978